l10n_it_intrastat_statement/models/intrastat_statement_purchase_section.py

```diff
@@ -131,8 +131,8 @@ def _prepare_statement_line(self, inv_intra_line, statement_id=None):
 
         res.update({
             'transaction_nature_id': transaction_nature_id.id,
-            'weight_kg': round(inv_intra_line.weight_kg) or 0,
-            'additional_units': round(inv_intra_line.additional_units) or 0,
+            'weight_kg': round(inv_intra_line.weight_kg) or 1,
```

Comment:
E se `inv_intra_line.weight_kg` fosse 0 sarebbe corretto mettere `1`?

Reply:
bisognerebbe distinguere tra merce e servizi, che dici, è possibile? 
Se è possibile fare questa distinzione opto per questo tipo di controllo. altrimenti come dici correttamente tu, se il valore, prima dell'arrotondamento è zero, lo mantengo.

Reply:
In effetti non lo so

Reply:
stavo pensando.. a scoppio molto ritardato... :)
i servizio non hanno peso..  può esistere una riga di merce con un peso uguale a zero?

Reply:
Se sono servizi dovrebbero andare nell'apposita sezione, in cui non è richiesta la valorizzazione del peso.
Questa parte di codice sta popolando la sezione dedicata alle merci

Reply:
esattamente, quindi a meno che ci sia un caso in cui i valori debbano essere effetti uguali a zero, la modifica della PR dovrebbe andare bene così. 
@alessandrocamilli ti viene in mente qualche casistica?

Reply:
casi di merce dove il peso deve essere zero? a me non viene in mente nulla :-)

Reply:
nemmeno a me a dire il vero :) @eLBati  mandiamo avanti, o hai ancora perplessità?

Reply:
No, grazie mille. Per me potete procedere